app/enterprise/1.3-x/kong-cloud.md

```diff
@@ -1,12 +1,74 @@
 ---
 title: Kong Cloud
-toc: false
+toc: true
 ---
 
-## Kong Cloud Proxy TLS
+# Kong Cloud Overview
+Kong Cloud is our managed Kong Enterprise offering, based on the proven Kong Enterprise Gateway (Kong API Gateway). Fully automated, forget about scaling, data persistence, high availability configurations, and failure recovery. [Start a free trial](https://konghq.com/products/kong-enterprise/free-trial) today and end your operational worries.
 
-Kong Cloud enforces HTTPS for all of the services that it operates, including Admin API, Kong Manager, and Kong Developer Portal. Kong Cloud does not enforce HTTPS for traffic destined for the customer's proxy, since certificate management and associated domains are under the customer's control. To enforce HTTPS on upstream traffic, use the [certificate](https://docs.konghq.com/enterprise/{{page.kong_version}}/admin-api/#certificate-object) and [SNI](https://docs.konghq.com/enterprise/{{page.kong_version}}/admin-api/#sni-object.) objects through the Admin API.
+# Kong Cloud  differences from on-premise
+Kong Cloud, while being our hosted Kong Enterprise solution, has a few key differences from a self-hosted, on-premise Kong Enterprise cluster.
 
-For non-proxy traffic, Kong Cloud is the terminus for the request, and Kong controls the protocol and shape of traffic carefully. Additionally, Kong generally considers non-proxy traffic to be sensitive (e.g., Admin API requests, login credentials to Kong Manager). Kong Cloud accomplishes enforcement through HTTP → HTTPS redirects, and the use of the HSTS response header. Because Kong Cloud controls the domain hosting the endpoints for these services, e.g., `https://manager-client.kong-cloud.com`, Kong Cloud maintains the TLS certificate for this service since Kong owns the domain.
+## Security
+By default, RBAC is enabled on both the Kong Manager and Kong Gateway Admin API to prevent unauthorized access to your Kong Enterprise cluster.
```

Comment:
For consistency, Kong Manager has no article. The Admin API should be distinct from the Gateway, and also clearly identified as the Enterprise version.

```suggestion
By default, RBAC is enabled on both Kong Manager and the Kong Enterprise Admin API to prevent unauthorized access to your Kong Enterprise cluster.
```